CHANGES.md

```diff
@@ -711,7 +711,7 @@ Bug Fixes:
 * CLOUDSTACK-1391 EventBus is not getting injected after javelin merge
 * CLOUDSTACK-1383 Deploying basic zone on 4.1 fails in NPE
```

Comment:
@sureshanaparti I think we should remove the CHANGES file - nobody maintains that.

Reply:
I would add that changes that were done when the branch was called 'master' were not done to 'main'. I agree on removing the file.